.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@sanadwj I'd suggest adding the idea folder to gitignore file, those are the settings for rubymine project and shouldn't be here

Reply:
Wierd it's in .gitignore, I will delete it.